app/src/main/java/com/facebook/encapp/BufferDecoder.java

```diff
@@ -113,6 +113,7 @@ public String start() {
         }
 
         Size res = SizeUtils.parseXString(mTest.getInput().getResolution());
+        // TODO(chema): this assumes 4:2:0 subsampling, and therefore YUV
         mRefFramesizeInBytes = (int) (res.getWidth() * res.getHeight() * 1.5);
```

Comment:
This should look at the color format and act accordingly (e.g. to be able to handle 444 or 10bit or whatever).

Reply:
The comment is unnecesary, instead the implementation should be fixed to use the raw format provided (as much as can be read from the format).

Reply:
> This should look at the color format and act accordingly (e.g. to be able to handle 444 or 10bit or whatever).
> The comment is unnecesary, instead the implementation should be fixed to use the raw format provided (as much as can be read from the format).

Fully agree (except the 10-bit part, which I doubt we'll have cycles for). 

In the meanwhile, I think having a TODO comment is a good thing.

Reply:
ok.